spec/controllers/organisers_controller_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+
+RSpec.describe OrganisersController, type: :controller do
+	describe 'Organisers Controller GET Tests' do
+		before(:each) do
+			@organiser = create(:user)
+		end
+
+		it 'should get the organiser dashboard' do
+			edition = create(:edition)
+			login_user(@organiser)
+			get :dashboard, user: @organiser
+			expect(response).to have_http_status(200)
+		end
+
+		it 'should get organiser dashboard with "final_project_evaluation"' do
+			edition = Edition.create(name: "test",
+				start_date: 100.days.ago,
+				end_date: (10.days.ago),
+				registration_started_at: 200.days.ago,
+				registration_closed_at: 185.days.ago)
+			login_user(@organiser)
+			get :dashboard, user: @organiser
+			expect(response).to have_http_status(200)
+		end
+
+		it 'should get organiser dashboard with "ongoing_projects"' do
+			edition = Edition.create(name: "test",
+				start_date: 100.days.ago,
+				end_date: (10.days.from_now),
+				registration_started_at: 200.days.ago,
+				registration_closed_at: 185.days.ago)
+			login_user(@organiser)
+			get :dashboard, user: @organiser
+			expect(response).to have_http_status(200)
+			# Something like this would be better
+			# expect(response).to render_template(file: 'app/views/organisers/ongoing_projects.slim')
+		end
+
+		it 'should get index page with some organisers' do
+			login_user(@organiser)
+			get :index
+			expect(response).to have_http_status(200)
+			expect(assigns(:pending_organisers)[0]).to eq(@organiser)
+		end
+
+		it 'should not get problematic projects' do
+			edition = create(:edition)
+			mentor = create(:mentor, last_activity_at: 2.days.ago)
+			mentee = create(:mentee, is_missing: false, last_activity_at: 2.days.ago)
+			project = create(:project, language: "", mentor_id: mentor.id, mentee_id: mentee.id, edition_id: edition.id)
+			login_user(@organiser)
+			get :problematic_projects
+			expect(assigns(:projects)[0]).to be(nil)
+		end
+
+		it 'should get problematic projects' do
+			edition = create(:edition)
```

Comment:
you can initialize records that you use in several tests using `let`. More explanation eg https://www.reddit.com/r/ruby/comments/2bcz12/newbie_to_rspec_why_use_let_instead_of_a_local/ but let me know if you have questions about it

Reply:
I changed all possible variables to 'let'